upi/vsphere/README.md

```diff
@@ -1,59 +1,47 @@
-# Pre-Requisites
+# Installing OpenShift on VMware
 
-* terraform
-* jq
+The terraform examples provided require a AWS account.  AWS is used
+for its API interface to network load balancing and DNS.
+The `./modules/rhcos_virtual_machines` can be used independently if so desired.
 
-# Build a Cluster
+We will begin with using `openshift-installer` as each terraform example 
+starts with creating the ignition files.
 
-1. Create an install-config.yaml.
-The machine CIDR for the dev cluster is 139.178.89.192/26.
+## Create ignition files for OpenShift RHCOS virtual machines
+
+You can create a `install-config.yaml` file by copying the example provided:
 
 ```
```

Comment:
nit: doesn't really add much in the way of syntax highlighting in most cases, but it's nice to explicitly declare this to be a shell block with ```sh.